src/frontend/src/routes/(app)/activity/+page.ts

```diff
@@ -0,0 +1,10 @@
+import { type RouteParams } from '$lib/utils/nav.utils';
+import type { LoadEvent } from '@sveltejs/kit';
+import type { PageLoad } from './$types';
+
+// We reset the data because the "activity" route operates without a network or token selected.
+export const load: PageLoad = (_$event: LoadEvent): RouteParams => ({
```

Comment:
Not fore this PR but, in another one, if you can refactor this to extract a function it would be nice. We have the same loader in three different routes now.

Reply:
> P.S. I will extract the function in another PR

I wrote the same [above](https://github.com/dfinity/oisy-wallet/pull/3352#issuecomment-2457394871) ahahaha!

Reply:
Done:

- PR #3362